src/main/java/org/antipathy/mvn_scalafmt/FormatMojo.java

```diff
@@ -26,23 +28,15 @@
     private boolean skipTestSources;
     @Parameter(property = "format.skipmain", defaultValue = "false")
     private boolean skipSources;
-    @Parameter(defaultValue = "${project.build.sourceDirectory}", required = true)
+    @Parameter(defaultValue = "${project.build.sourceDirectory}/../scala", required = true)
```

Comment:
I'll be honest, I'm not overly keen on the relative path in here.  But I don't see an alternative to get the same result.